src/main/webapp/components/InternationalCohorts.js

```diff
@@ -32,7 +32,10 @@ export const InternationalCohorts = hh(class InternationalCohorts extends Compon
     let questions = [];
 
     questions.push({
-      question: span({}, ["Are samples or individual-level data sourced from a country in the European Economic Area? ", span({ className: "normal" }, ["[provide link to list of countries included]"])]),
+      question: span({}, [
+        "Are samples or individual-level data sourced from a country in the European Economic Area? ", 
+        a({ href:"https://www.imf.org/external/pubs/ft/fandd/2014/03/europeaneconomicarea.htm", target: "_blank", className: "normal" }, ["(List of member states of European Economic Area)"])
```

Comment:
nit: I think this string array can just be a string (here and in other places too). Looking at the [documentation](https://www.npmjs.com/package/react-hyperscript-helpers), this can be an array, boolean, string, number, etc. I do know that when I tried that with a `div`, it threw an error (or I messed up ... very likely) and expected an array, so I think the docs might be a little fuzzy on that detail.
```suggestion
        a({ href:"https://www.imf.org/external/pubs/ft/fandd/2014/03/europeaneconomicarea.htm", target: "_blank", className: "normal" }, "(List of member states of European Economic Area)")
```

Reply:
oh nice! didn't know that. I'll change it, thanks!